src/test/java/net/snowflake/client/core/arrow/VarCharConverterTest.java

```diff
@@ -107,4 +107,23 @@ public void testGetBoolean() throws SFException {
 
     vector.close();
   }
+
+  @Test
+  public void testGetDate() throws SFException {
```

Comment:
I named the test testGetDate() even though it's actually testing toDate() to fit with the rest of the test names in this class which do it the same way. If this is not wanted, I can easily change the test name.